persistence/sql/migrations/sql/20241106142200000002_identities.autocommit.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE INDEX IF NOT EXISTS identity_credential_identifiers_ici_nid_i_idx
+    ON identity_credential_identifiers (identity_credential_id ASC, nid ASC, identifier ASC);
```

Comment:
@alnr i changed the index structure to use `identity_credential_id` as the prefix (it is also the highest cardinality) and then we can use it both for FK and the query credentials